apps/web/src/components/Messages/MessageHeader.tsx

```diff
@@ -16,14 +17,23 @@ interface MessageHeaderProps {
 const MessageHeader: FC<MessageHeaderProps> = ({ profile }) => {
   const router = useRouter();
   const [following, setFollowing] = useState(true);
+  const unsyncProfile = useMessageStore((state) => state.unsyncProfile);
```

Comment:
I thought I created a bug with my changes, but it looks like the behaviour on prod is the same. Following a user should move them from the `Requested` to the `Following` tab. Now it actually does that.